<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HazardsForm</class>
 <widget class="QWidget" name="HazardsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1631</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hl_hazards" stretch="1,1">
     <item>
      <widget class="QTreeWidget" name="hazards_types">
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Код</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тип вредности (Приложение 2)</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="hazards_factors">
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Код</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Фактор вредности (Приложение 1)</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl_summary">
     <item>
      <layout class="QFormLayout" name="fl_types_summary">
       <item row="0" column="0">
        <widget class="QLabel" name="types_summary_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Выбраны типы:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="types_summary">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="fl_factors_summary">
       <item row="0" column="0">
        <widget class="QLabel" name="factors_summary_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Выбраны факторы:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="factors_summary">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fl_hazards_btns">
     <item>
      <spacer name="horizontal_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_btn">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить и закрыть (Enter)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>hazards_types</tabstop>
  <tabstop>hazards_factors</tabstop>
  <tabstop>save_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
